### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by olenagerasimova**
Bumped our dependencies and made corresponding corrections.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@artipie/contributors please review, I cannot assign PRs in this repo

File: npm-proxy-adapter/pom.xml [hunk 1]
```diff
@@ -56,12 +56,12 @@ SOFTWARE.
     <dependency>
       <groupId>com.artipie</groupId>
       <artifactId>asto</artifactId>
-      <version>0.23.2</version>
+      <version>0.24</version>
```
Comment (olegmoz) [inline npm-proxy-adapter/pom.xml:59]: @olenagerasimova shouldn't we bump it to `0.28` which is latest version?

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

**Comment by olegmoz** [comment]
@olenagerasimova please also note that CI checks are failing for some reason

**Comment by olenagerasimova** [comment]
@olegmoz all fixed, please, merge and release

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks

**Comment by olegmoz** [comment]
@yegor256 please touch `.rultor` so we can release

**Comment by yegor256** [comment]
@olegmoz done

**Comment by olegmoz** [comment]
@yegor256 thanks!

**Comment by olegmoz** [comment]
@rultor release, tag=`0.6.2`

**Comment by 0crat** [comment]
Job `gh:artipie/npm-adapter#118` is not assigned, can't get performer

**Comment by olegmoz** [comment]
@rultor release, tag=`0.6.2`

**Comment by olegmoz** [comment]
@rultor release, tag=`0.6.2`

**Comment by olegmoz** [comment]
@rultor release, tag=`0.6.2`

**Comment by olegmoz** [comment]
@olenagerasimova release complete

**Pull request merged**